.gitmodules

```diff
@@ -16,3 +16,6 @@
 [submodule "subxt"]
 	path = subxt
 	url = https://github.com/Phala-Network/substrate-subxt.git
+[submodule "standalone/prouter/i2pd"]
+	path = standalone/prouter/i2pd
```

Comment:
How about moving it to the Phala official repo?

Reply:
Absolutely, but we can do it after this PR (or right before the final merging).